.gitignore

```diff
@@ -21,3 +21,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+.env
+.idea
```

Comment:
I think you should add .idea to your global .gitignore

Reply:
.idea moved